astropy/io/fits/hdu/compressed.py

```diff
@@ -414,7 +414,7 @@ def __init__(self, data=None, header=None, name=None,
         data : array, optional
             Uncompressed image data
 
-        header : Header instance, optional
+        header : :class:`~astropy.io.fits.Header`, optional
```

Comment:
do we need the `:class:` part? the link is bad enough in the docstring when one looks at it in an interpreter

Reply:
Fixed